### PR TITLE
GPII-4479: Update to work with newer versions of Infusion.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [GPII-4479](https://issues.gpii.net/browse/GPII-4479) for details.

### CODE REVIEW

File: tests/js/common/lib/check-potentia-grades.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+(function (fluid) {
+    "use strict";
+    if (!fluid.identity) {
+        fluid = require("infusion");
+    }
+
+    var gpii = fluid.registerNamespace("gpii");
+
+    require("../../../../");
```
Comment (amb26) [inline tests/js/common/lib/check-potentia-grades.js:13]: Generally nicer to write fluid.require("%gpii-json-schema") if you can

Comment (the-t-in-rtf) [reply tests/js/common/lib/check-potentia-grades.js]: Thanks for catching that, it's fixed.

**Comment by amb26** [comment]
Hang invoking testem - ok to test

**Comment by the-t-in-rtf** [comment]
Error late in the VM creation process, which resulted in a usable node but no grunt.  So the VM creation failed, as did the linting.  The actual tests passed.  Ok to test again.

**Comment by the-t-in-rtf** [comment]
Same provisioning error, will try to reproduce locally.

**Comment by the-t-in-rtf** [comment]
Passes locally using the same steps we use in CI.

**Comment by the-t-in-rtf** [comment]
Ok to test again.

**Comment by the-t-in-rtf** [comment]
@amb26, per my recent goof with gpii-handlebars I went ahead and nudged all dependencies.  As soon as this passes CI, IMO we are good to merge and move on to fixing gpii-express-user.

**Pull request merged**